waveform-django/static/debug_toolbar/css/print.css

```diff
@@ -0,0 +1,3 @@
+#djDebug {
```

Comment:
I think this is part of my static files commit which shouldn't be included here... you should be able to remove it by:
`git rebase -i HEAD~4` then dropping my commit [static files] followed by a `git push -f`

Reply:
I rebased the branch as you requested, but it's still a little unclear to me as to why the main branch has these CSS issues when ran locally but the live website works just fine.

Reply:
Yeah it's weird, the server houses static files weirdly due to the nginx setup I still have to figure out how to get it to work on both platforms!